DEVELOPMENT.md

```diff
@@ -160,9 +160,9 @@ Command line arguments
 ----------------------
 
 Note that different set of programs is used in different
-DeviceModes. Three stages: *pmem-ns-init*, *pmem-vgm*,
-*pmem-csi-driver* run in DeviceMode:LVM. Only *pmem-csi-driver* runs
-in DeviceMode:Direct.
+device modes. Three stages: *pmem-ns-init*, *pmem-vgm*,
+*pmem-csi-driver* run in lvm device mode. Only *pmem-csi-driver* runs
```

Comment:
"lvm" is an acronym. It must be in capital letters here. Might also have to fixed elsewhere, but only in plain text. If it is used as a keyword or in a path name, it can be lowercase (similar to PMEM-CSI).

For example,

    `direct` or `lvm`: one uses direct device mode, the other LVM device mode

looks good to me.